### DISCUSSION TITLE
radarr path

### DISCUSSION THREAD

**Discussion by foXaCe** [jesec/flood#208] [Q&A]
hello, with sonarr flood qbit is working fine. On the other hand with radarr it does not go in the "radarr" folder it stays in / data. Why ?

**Comment by jesec** [comment]
As this is a support/question, I am moving it to the discussions. 

Note that this project does not _officially_ provide support to downstream integrations. It is recommended to seek help from other community members via Discord when such issues arise. 

You should specify destination if you want to download contents to a different path.

![image](https://user-images.githubusercontent.com/11585141/108401273-24336000-7257-11eb-857e-3ead22dec882.png)
